mne/channels/layout.py

```diff
@@ -375,6 +375,10 @@ def find_layout(info, ch_type=None, exclude='bads'):
     exclude : list of string | str
         List of channels to exclude. If empty do not exclude any.
         If 'bads', exclude channels in info['bads'] (default).
+    on_failure : None | 'generate'
+        What to do if no layout is found. If None (default), None is returned.
+        If 'generate', a custom layout is generated from all the channel
+        locations.
```

Comment:
It is an API change of sort. Before, if the layout could not be found, it would return None and the predefined layouts are not using the exact channel locations.